lib/MusicBrainz/Server/Data/AreaType.pm

```diff
@@ -6,11 +6,13 @@ use MusicBrainz::Server::Entity::AreaType;
 use MusicBrainz::Server::Data::Utils qw( load_subobjects );
 
 extends 'MusicBrainz::Server::Data::Entity';
-with 'MusicBrainz::Server::Data::Role::EntityCache' => { prefix => 'area_type' };
+with 'MusicBrainz::Server::Data::Role::EntityCache';
 with 'MusicBrainz::Server::Data::Role::SelectAll';
 with 'MusicBrainz::Server::Data::Role::OptionsTree';
 with 'MusicBrainz::Server::Data::Role::Attribute';
 
+sub _type { 'area_type' }
```

Comment:
`_type` is the key into `%ENTITIES` and is required to look up the cache id/prefix now.

I'm not a fan of adding all these, since I think we store them in the Entity classes too. I tried adding a method that looks them up based on the package name, but it caused some test failures I couldn't figure out, so I gave in.